.circleci/config.yml

```diff
@@ -113,147 +125,108 @@ build_and_deploy: &build_and_deploy
-                cat ./env_list.txt
-                grep -E "${env_pattern}" ./env_list.txt > ./matching_env_list.txt || true
-                # If the file is NOT empty
-                if [ -s "./matching_env_list.txt" ]
-                then
-                  echo "Start to delete multi-dev..."
-                  while read env_name; do
-                    terminus multidev:delete --delete-branch $CIRCLE_PROJECT_REPONAME.$env_name -y
-                    echo "Deleted multi-dev: $env_name"
-                  done <./matching_env_list.txt
-                  echo "Done deleting multi-dev"
-                  rm -f ./matching_env_list.txt
-                else
-                  echo "No multi-dev site found matching the story ID."
-                fi
-              fi
-            fi
-            echo "Waiting for sync code workflow to finish..."
-            until [ $(terminus workflow:list --fields status,env,workflow --format string -- $CIRCLE_PROJECT_REPONAME | egrep -i "$PANTHEON_ENV\s+Sync code" -m 1 | egrep -i -o "^\w+") == succeeded ]; do
-              sleep 30
-            done
-            echo "Rebuilding cache..."
-            terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cr
-            echo "Running database updates..."
-            terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- updb -y
-            echo "Importing config..."
-            terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cim -y -vvv
-            echo "Running cron..."
-            terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- core:cron -y
-            echo "Rebuilding cache..."
-            terminus drush $CIRCLE_PROJECT_REPONAME.$PANTHEON_ENV -- cr
-          else  # All other commits deploy to a multidev
-            # Set Pantheon environment name
-            if [[ $CIRCLE_BRANCH =~ ^dependabot\/ ]] && [[ $CIRCLE_PULL_REQUEST =~ \/pull\/([0-9]+)$ ]]; then
-              echo "export PANTHEON_ENV=bot-${BASH_REMATCH[1]}" >> $BASH_ENV && source $BASH_ENV
+            JIRA_STORY_ID=""
+            # Simple merge
+            # Example: Merge pull request #35 from eGovPDX/powr-152
+            if [[ $lowercase_last_log =~ (^merge pull request #[0-9]* from egovpdx\/)([^,]*) ]]; then
+              echo "Found story ID from merge commit message: '${BASH_REMATCH[2]}'";
+              JIRA_STORY_ID=${BASH_REMATCH[2]}
+            # Squash and Merge on other feature branches
+            # Example: POWR-3058 Delete all related multi-devs when a branch is merged
+            elif [[ $lowercase_last_log =~ ^(powr-[0-9]+)[[:space:]]+.+$ ]]; then
+              echo "Found story ID from squash and merge commit message: '${BASH_REMATCH[1]}'";
+              JIRA_STORY_ID=${BASH_REMATCH[1]}
             else
-              echo "export PANTHEON_ENV=$CIRCLE_BRANCH" >> $BASH_ENV && source $BASH_ENV
+              echo "Cannot find JIRA story ID from the last commit message: $lowercase_last_log"
+              exit 1
```

Comment:
Am I correct in thinking this will fail our build if there isn't a multidev to delete?

Reply:
Will that cause issues (build fail) when we merge the `release` branch into `master`

Reply:
Nevermind... found the if `dev` or `release` at the start of this code 😄